packages/forklift-console-plugin/jest.config.ts

```diff
@@ -5,20 +5,24 @@ import type { Config } from '@jest/types';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { compilerOptions } = require('./tsconfig.json');
 
+const moduleNameMapper = {
+  '\\.(css|less|scss|svg)$': '<rootDir>/src/__mocks__/dummy.ts',
+  '@console/*': '<rootDir>/src/__mocks__/dummy.ts',
+  '@openshift-console/*': '<rootDir>/src/__mocks__/dummy.ts',
+  'react-i18next': '<rootDir>/src/__mocks__/react-i18next.ts',
+  'legacy/src/(.*)$': '<rootDir>/../legacy/src/$1', // remove when using rollup
+  'common/src/(.*)$': '<rootDir>/../common/src/$1', // remove when using rollup
+  ...pathsToModuleNameMapper(compilerOptions.paths, {
+    prefix: '<rootDir>/',
+  }),
+};
```

Comment:
I did it this way with no paths references needed:

```js
  moduleNameMapper: {
    '\\.(css|less|scss|svg)$': '<rootDir>/src/__mocks__/dummy.ts',
    '@console/*': '<rootDir>/src/__mocks__/dummy.ts',
    '@openshift-console/*': '<rootDir>/src/__mocks__/dummy.ts',
    'react-i18next': '<rootDir>/src/__mocks__/react-i18next.ts',

    // teach jest about the paths defined in tsconfig.json
    ...pathsToModuleNameMapper(compilerOptions.paths, {
      prefix: '<rootDir>/',
    }),

    // teach jest about the project references defined in tsconfig.json
    '@kubev2v/common/(.*)': '<rootDir>/../common/src/$1',
    '@kubev2v/legacy/(.*)': '<rootDir>/../legacy/src/$1',
    '@kubev2v/mocks/(.*)': '<rootDir>/../mocks/src/$1',
  },
```

Reply:
looks the same to me, what am I missing ?

Reply:
You have:
```js
  'legacy/src/(.*)$': '<rootDir>/../legacy/src/$1', // remove when using rollup
  'common/src/(.*)$': '<rootDir>/../common/src/$1', // remove when using rollup
```

while I referenced the package names directly.  And I did not have the `@kubev2v` references in my tsconfig.json.